proxy.go

```diff
@@ -158,7 +167,8 @@ func (ph ProxyHandler) proxyRequest(w http.ResponseWriter, req *http.Request, au
 			req.Body = ioutil.NopCloser(rd)
 			resp, err = auth.do(req, ph.transport)
 			if err != nil {
-				http.Error(w, err.Error(), http.StatusInternalServerError)
+				log.Printf("[%d] Error forwarding request (with auth): %v", id, err)
```

Comment:
Unfortunately github does not allow me to comment on lines outside the diff context.

5 lines above is an error being logged, but it does not have the id, and it also does not write an error or return. Not sure if being unable to seek is a terminal error, but perhaps at least log the id now that you have it.

Reply:
Done, thanks for catching that